a3p-integration/proposals/p:upgrade-19/package.json

```diff
@@ -10,13 +10,14 @@
       "testing/test-upgraded-board.js testUpgradedBoard",
       "vats/upgrade-agoricNames.js agoricNamesCoreEvals/upgradeAgoricNames",
       "testing/add-USD-OLIVES.js agoricNamesCoreEvals/addUsdOlives",
-      "testing/publish-test-info.js agoricNamesCoreEvals/publishTestInfo"
+      "testing/publish-test-info.js agoricNamesCoreEvals/publishTestInfo",
+      "vats/upgrade-mintHolder.js upgrade-mintHolder USDC_axl USDT_grv DAI_axl DAI_grv stATOM USDC_grv ATOM USDT_axl USDC BLD IST"
```

Comment:
Initially, I used an assertion on the core-eval to verify that a mintHolder contract kit existed for each asset label. However, because IST was the last item in the list, the test passed even when a mintHolder was missing.

To ensure that the core-eval process doesn't fail completely while still identifying issues, I added a condition (see [here]()) . If the condition fails, an error is logged specifying the asset that couldn't be upgraded. However, the process continues without throwing an exception.